backend/routes/home/index.js

```diff
@@ -6,6 +6,7 @@ const usersInfo = require('../../userInfo.json')
 
 /* Successfully inquiry of authentication */
 router.get('/', (req, res) => {
+  console.log('GET /')
```

Comment:
ここはすでに/homeの配下なので、下記の方が良いと思います！
```suggestion
  console.log('GET /home')
```